### PR TITLE
Xcs

### PR DESCRIPTION
**Pull request by msaroshali**
Fixed and added some hyperlinks.

### CODE REVIEW

File: content/en/load-testing/Quick Start/40-run-a-test.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ com.xceptance.xlt.loadtests.TVisit.users = 1
 com.xceptance.xlt.loadtests.TVisit.arrivalRate = 500
 ```
 
-In this case we run a test for 1 hour with a 5 minute ramp-up period, and a 1 minute shutdown phase. The shutdown phase will not turn up in our measurements, but ramp-up does. You can exclude it later if desired. Total runtime is 1h 5min (measurement period + shutdown period).
+In this case we run a test for 1 hour with a 5 minute [ramp-up](../../glossary/#ramp-up-period-xlt) period, and a 1 minute [shutdown](../../glossary/#shutdown-period-xlt) period. The shutdown phase will not turn up in our measurements, but ramp-up does. You can exclude it later if desired. Total runtime is 1h 5min ( [measurement](../../glossary/#measurement-period-xlt) period + shutdown period).
```
Comment (js-xc) [inline content/en/load-testing/Quick Start/40-run-a-test.md:34]: As the Glossary is for "ramp-up _period_", let's link the whole `[ramp-up period]`. Same for shutdown/measurement.

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 2]
```diff
@@ -8,19 +8,35 @@ type: docs
 description: >
   You can find a list of available commands below.
 ---
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --c` or `mastercontroller --commands` displays all the available options and a brief guide about their usage.
```
Comment (js-xc) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:14]: I think what you meant is `mastercontroller --help`. This is what the commands option is used for: https://xltdoc.xceptance.com/load-testing/manual/310-test-execution/#non-interactive-mode-scripted-commands

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 3]
```diff
@@ -8,19 +8,35 @@ type: docs
...
-6. `-faf`
-7. `-noDownload`
-8. `-o`
-9. `--only-download`
-10. `-pf`
-11. `report`
-12. `sequential`
-13. `testPropertiesFile`
-14. `timezone`
+- `-auto` is to be [used](../310-test-execution/#auto-mode) for running a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands).
```
Comment (js-xc) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:29]: Might be misleading to link to both. The `-auto` option is starting mastercontroller in auto mode (https://xltdoc.xceptance.com/load-testing/manual/310-test-execution/#auto-mode), which is also non-interactive but you have no chance to define the commands that should be run, it just uses the default command sequence.

Comment (msaroshali) [reply content/en/load-testing/Manual/800-master-controller-cheatsheet.md:29]: Alright, should I be removing the link for auto-mode?

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md (hunk 2 above)
Comment (msaroshali) [reply content/en/load-testing/Manual/800-master-controller-cheatsheet.md:14]: --help is working the same as --c or --commands. Both display the options and their usage if used in that way. But I'll update it as --help.

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+title: "Master Controller Cheat-Sheet"
+linkTitle: "Cheat-Sheet MasterController"
```
Comment (js-xc) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:3]: spelling: cheat sheet (no hyphen), mastercontroller (one word, lowercase, that's how it should be throughout the doc)

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+---
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands). by executing all commands automatically.
+
+### Example(Windows PowerShell)
+
+- This would display all the available command options
```
Comment (js-xc) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:19]: see comments [here](https://github.com/Xceptance/xlt-documentation/pull/100) for shell example, same same

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+- `-noDownload` to be used with auto mode only. Test results are not downloaded. The option is ignored in sequential/[non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands).
+
+- `-o <dir>`  stores test results that have been downloaded in the directory specified. 
+
+- `--only-download <dataTypeList>` restricts download to only the given result data types in a comma-separated list. Applicable in non-interactive mode only.
+
+- `-pf <file>` can be employed to utilize a separate properties file, which overrides the values in file `mastercontroller.properties`.
+
+- `report` will generate a report after downloading test results. it will not be effective in commands mode.
```
Comment (js-xc) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:49]: spelling, "It" (uppercase for new sentence? Yes, it's the little things, we're in QA, we have a reputation)

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+- `sequential` used for running test cases sequentially(one after the other).
```
Comment (js-xc) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:51]: space before bracket (see above, the QA reputation)

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+- `testPropertiesFile <filename>` can be used for specifying properties file for the test-run.  
+
+- `timezone <timezoneID>` [overrides](../../../release-notes/4.2.x/#custom-time-zone-for-reports-1536) user's default timezone, while generating a test report.
```
Comment (js-xc) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:55]: see comment on https://github.com/Xceptance/xlt-documentation/pull/100 (you can make the change in either branch and just add the link on both then, there are no link validity checks yet)

File: content/en/load-testing/_index.md [hunk 9]
```diff
@@ -4,7 +4,7 @@ linkTitle: "Load Testing"
 
 type: docs
 
-weight: 10
+weight: 50
```
Comment (js-xc) [inline content/en/load-testing/_index.md:7]: unnecessary change, please revert

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md (hunk 8 above)
Comment (msaroshali) [reply content/en/load-testing/Manual/800-master-controller-cheatsheet.md:57]: alright. linking it to the text I added in the create_report branch http://localhost:1313/load-testing/manual/540-report-options/#setting-a-custom-time-zone

File: content/en/load-testing/_index.md (hunk 9 above)
Comment (js-xc) [reply content/en/load-testing/_index.md:7]: If I can still see the change here it's not reverted yet

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
+---
+title: "Mastercontroller Cheat Sheet"
+linkTitle: "Cheat Sheet Mastercontroller"
+
+weight: 800
+type: docs
+
+description: >
+  You will find the usage instructions and a list of available commands below.
```
Comment (randolph-straub) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:9]: Maybe a short summary of the purpose of the master controller would be nice. Maybe something like "primary tool to run the load tests", "can be used interactive or in auto mode"... something like that...

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+>>./mastercontroller.cmd --help 
+```
+- This would make a load test run in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands), executing commands automatically in combination with [embedded mode](../310-test-execution/#embedded-mode).
+```powershell 
+cd <XLT>\bin
+>>./mastercontroller.cmd -auto -embedded -comment "Test Run" 
+```
+
+## List of Command Options
+- `-auto` is to be used for running a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands).
```
Comment (randolph-straub) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:31]: A short summary of which steps the auto mode include and where it stops would be nice... and a hint to the -report option might be helpful

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+
+- `-embedded` utilizes a single embedded agent controller. It makes the test run in [embedded mode](../310-test-execution/#embedded-mode).
+
+- `-faf` Usage deprecated. Recommended to use `-auto` instead.
+
+- `-noDownload` to be used with auto mode only. Test results are not downloaded. The option is ignored in sequential/[non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands).
+
+- `-o <dir>`  stores test results that have been downloaded in the directory specified. 
+
+- `--only-download <dataTypeList>` restricts download to only the given result data types in a comma-separated list. Applicable in non-interactive mode only.
```
Comment (randolph-straub) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:47]: What are the possible result data type values a user could pass as <dataTypeList>? A short listing would be nice.

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+
+- `-pf <file>` can be employed to utilize a separate properties file, which overrides the values in file `mastercontroller.properties`.
+
+- `report` will generate a report after downloading test results. It will not be effective in commands mode.
```
Comment (randolph-straub) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:51]: A hint to the -auto option is missing.

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+
+- `sequential` used for running test cases sequentially (one after the other).
+
+- `testPropertiesFile <filename>` can be used for specifying properties file for the test-run.  
+
+- `timezone <timezoneID>` [overrides user's default timezone,](../540-report-options/#setting-a-custom-time-zone) while generating a test report.
```
Comment (randolph-straub) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:57]: How does a valid timezoneID looks like? Maybe examples for valid timezoneID's that are mostly used could be helpful.

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands) by executing all commands automatically.
+
+### Example
+
+- For displaying all the available command options you can run
+```powershell
```
Comment (randolph-straub) [inline content/en/load-testing/Manual/800-master-controller-cheatsheet.md:20]: Do we plan to have Linux examples later too?

**Review by randolph-straub — COMMENTED** [review]
Basically this is what I was hoping for. Great start!

File: content/en/load-testing/How-Tos/master-controller-cheatsheet.md [hunk 16]
```diff
@@ -0,0 +1,57 @@
...
+
+- For displaying all the available command options you can run
+```powershell
+cd <XLT>\bin 
+>>./mastercontroller.cmd --help 
+```
+- This would make a load test run in [auto mode](../../manual/310-test-execution/#auto-mode), executing commands automatically in combination with [embedded mode](../../manual/310-test-execution/#embedded-mode).
+```powershell 
+cd <XLT>\bin
+>>./mastercontroller.cmd -auto -embedded -comment "Test Run"
```
Comment (js-xc) [inline content/en/load-testing/How-Tos/master-controller-cheatsheet.md:27]: same here

File: content/en/load-testing/How-Tos/master-controller-cheatsheet.md [hunk 17]
```diff
@@ -0,0 +1,57 @@
...
+- `mastercontroller [<other options>]` runs in [interactive mode](../../manual/310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in non-interactive [auto mode](../../manual/310-test-execution/#auto-mode) by executing all commands automatically.
+
+### Example
+
+- For displaying all the available command options you can run
+```powershell
+cd <XLT>\bin 
+>>./mastercontroller.cmd --help
```
[the remark this answers is not in the document]
Comment (js-xc) [reply content/en/load-testing/How-Tos/master-controller-cheatsheet.md:22]: (actually I don't think we need the example for usage with `--help` as this is mentioned above in **Usage**... just keep the single example below)

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md (hunk 15 above)
Comment (js-xc) [reply content/en/load-testing/Manual/800-master-controller-cheatsheet.md]: Linux examples are kinda "standard" for the doc, however we could put Linux and Windows examples if you think that's helpful.

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md (hunk 12 above)
Comment (js-xc) [reply content/en/load-testing/Manual/800-master-controller-cheatsheet.md]: see https://xltdoc.xceptance.com/release-notes/5.7.x/#download-options-for-the-master-controller-in-non-interactive-mode - it's a new feature, so please add it to the doc, too (i.e. at the bottom of the page here https://xltdoc.xceptance.com/load-testing/manual/310-test-execution) and then link there

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md (hunk 11 above)
Comment (js-xc) [reply content/en/load-testing/Manual/800-master-controller-cheatsheet.md]: Default Steps: upload, start, download - please update this info in https://xltdoc.xceptance.com/load-testing/manual/310-test-execution/#auto-mode

File: content/en/load-testing/Manual/800-master-controller-cheatsheet.md (hunk 14 above)
Comment (msaroshali) [reply content/en/load-testing/Manual/800-master-controller-cheatsheet.md]: Created a section for timezone on a separate branch, under the following link: http://localhost:1313/load-testing/manual/540-report-options/#setting-a-custom-time-zone
The section also includes an example. Linking to that section for now.

File: content/en/load-testing/How-Tos/master-controller-cheatsheet.md [hunk 18]
```diff
@@ -0,0 +1,61 @@
...
+
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../../manual/310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [auto mode](../../manual/310-test-execution/#auto-mode) by executing all commands automatically.
+
+### Example
+
+#### This will make a load test run in [auto mode](../../manual/310-test-execution/#auto-mode), in combination with [embedded mode](../../manual/310-test-execution/#embedded-mode).
```
Comment (js-xc) [inline content/en/load-testing/How-Tos/master-controller-cheatsheet.md:22]: Please do NOT use a headline to make text bold.

File: content/en/load-testing/Manual/310-test-execution.md [hunk 19]
```diff
@@ -192,9 +192,9 @@ cd <XLT>/bin
 ./mastercontroller.sh -auto
 ```
 
-When the test suite is uploaded and the test started successfully, XLT automatically refreshes the agent status until the test finished. When the test finished, the results are downloaded.
+When the test suite is uploaded and the test has started successfully, XLT automatically refreshes the agent status until the test has finished. The results are downloaded, when the test finishes.
```
Comment (js-xc) [inline content/en/load-testing/Manual/310-test-execution.md:195]: There should be no comma in the last sentence.

File: content/en/load-testing/How-Tos/master-controller-cheatsheet.md [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+
+ This will make a load test run in [auto mode](../../manual/310-test-execution/#auto-mode), in combination with [embedded mode](../../manual/310-test-execution/#embedded-mode).
+##### Windows
+```powershell 
+cd <XLT>\bin
+./mastercontroller.cmd -auto -embedded -comment "Test Run" 
+```
+##### Linux
+```shell 
+cd <XLT>\bin
```
Comment (js-xc) [inline content/en/load-testing/How-Tos/master-controller-cheatsheet.md:30]: tiny mistake: path separator in Linux is `/`, not `\` (with a backslash it also does some weird code highlighting for shell, because `\b` is considered a special char)

File: content/en/load-testing/How-Tos/master-controller-cheatsheet.md [hunk 21]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+## List of Command Options
+- `-auto` is to be used for running a load test in [auto mode](../../manual/310-test-execution/#auto-mode). This option will automatically execute a number of [steps](../../manual/310-test-execution/#steps-executed-in-auto-mode) including upload of the test suite, starting the test, and download of the test results. It can also be used in combination with the `-report` option to create a report automatically on completion of the test.  
+
+- `-c <commandList>` or `--commands <commandList>` will execute commands in a comma-separated list, in the specified order and then quit. `abort`, `upload`,`report`,`start`, `download` are the supported commands that may be used.
+
+- `-comment <string>` sets a comment for your test run. 
+
+- `-D <property=value>` is used for [overriding or configuring](../490-environment-configuration/#configuration-via-command-line) a property in `mastercontroller.properties` file.
```
Comment (js-xc) [inline content/en/load-testing/How-Tos/master-controller-cheatsheet.md:41]: link does not work, please fix

File: content/en/load-testing/How-Tos/master-controller-cheatsheet.md [hunk 22]
```diff
@@ -0,0 +1,61 @@
...
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../../manual/310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --help` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [auto mode](../../manual/310-test-execution/#auto-mode) by executing all commands automatically.
+
+### Example
+
+ This will make a load test run in [auto mode](../../manual/310-test-execution/#auto-mode), in combination with [embedded mode](../../manual/310-test-execution/#embedded-mode).
+##### Windows
```
Comment (js-xc) [inline content/en/load-testing/How-Tos/master-controller-cheatsheet.md:23]: h4 (####) should be enough?

Comment (msaroshali) [reply content/en/load-testing/How-Tos/master-controller-cheatsheet.md:23]: There wasn't a noticeable difference between h4 and h3 at the first glance, but now that I looked closely, there is a little difference. Making corrections now.

File: content/en/load-testing/How-Tos/master-controller-cheatsheet.md [hunk 23]
```diff
@@ -0,0 +1,61 @@
...
+
+### Example
+
+ This will make a load test run in [auto mode](../../manual/310-test-execution/#auto-mode), in combination with [embedded mode](../../manual/310-test-execution/#embedded-mode).
+#### Windows
+```powershell 
+cd <XLT>\bin
+./mastercontroller.cmd -auto -embedded -comment "Test Run" 
+```
+##### Linux
```
Comment (js-xc) [inline content/en/load-testing/How-Tos/master-controller-cheatsheet.md:28]: h4 like "Windows" above
